gui/pages/user_add.go

```diff
@@ -76,15 +76,15 @@ func NewUserAddPage(controller Controller, model *model.SystemInstall) (Page, er
 
 	// Name
 	page.name, page.nameWarning, err = page.setSimilarWidgets(utils.Locale.Get("User Name"),
-		utils.Locale.Get("Must start with letter. Can use numbers, hyphens and underscores. Max %d characters.", user.MaxUsernameLength),
+		utils.Locale.Get("Must start with letter. Can use numbers, commas, - and _. Max %d characters.", user.MaxUsernameLength),
```

Comment:
So I didn't add the 'commas' originally because that is a field separator and didn't care to explain the nuances. If you are going to add it, it should be a character like the others.
We should again single quote all of the valid characters for clarity.
"Must start with letter. Can use numbers, ',', '-' and '_'. Max %d characters."

Reply:
@mdhorn I was trying to keep the rules msg similar to the warning message returned by the validation functions. E.g in IsValidUsername(). So should I be changing it everywhere to have single quotes?

Reply:
@reaganlo hmmm... maybe these rule strings belong in the non-UI class?
Or can they be constructed using the valid calls on known bad values to get the strings?
rule := IsValidUsername("") + IsValidUsername("super long string that is invalude")?

Reply:
After our discussion, I'm okay with what we have already in the pull request.